### PR TITLE
Updates the policy references post split of spec

### PR DESCRIPTION
**Pull request by emanic**
## Description

- Updates NetworkPolicy spec to remove PreDNAT, DoNotTrack, and ApplyOnForward. NetworkPolicy is namespaced and cannot select host endpoints. Per https://github.com/projectcalico/libcalico-go/pull/639/

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: master/reference/calicoctl/resources/globalnetworkpolicy.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Select a namespace in a GlobalNetworkPolicy in the standard selector by using
 value to compare against, e.g., `projectcalico.org/namespace == "default"`.
 See [NetworkPolicy]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/networkpolicy) for namespaced Network Policy.
 
-GlobalNetworkPolicy resources can be used to define network connectivity rules between groups of {{site.prodname}} endpoints and host endpoints, and
+GlobalNetworkPolicy resources can be used to define network connectivity rules between groups of Calico endpoints and host endpoints, and
```
Comment (gunjan5) [inline master/reference/calicoctl/resources/globalnetworkpolicy.md]: hmm shouldnt this be `{{site.prodname}}`?

File: master/reference/calicoctl/resources/globalnetworkpolicy.md [hunk 2]
```diff
@@ -55,7 +55,39 @@ spec:
 
 #### Spec
 
-{% include {{page.version}}/policyspec.md %}
+| Field          | Description                                                                                                                                           | Accepted Values | Schema                | Default |
+|----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------|-----------------------|---------|
+| order          | Controls the order of precedence. Calico applies the policy with the lowest value first.                                                              |                 | float                 |         |
```
Comment (gunjan5) [inline master/reference/calicoctl/resources/globalnetworkpolicy.md]: same here `Calico` -> `{{site.prodname}}`

File: master/reference/calicoctl/resources/globalnetworkpolicy.md (hunk 1 above)
Comment (emanic) [reply master/reference/calicoctl/resources/globalnetworkpolicy.md]: Thanks! I just fixed this

File: v3.0/reference/calicoctl/resources/networkpolicy.md [hunk 3]
```diff
@@ -6,13 +6,13 @@ no_canonical: true
...
 to a collection of endpoints that match a [label selector](#selector).
 
-NetworkPolicy is a namespaced resource. NetworkPolicy in a specific namespace
-only applies to [WorkloadEndpoints]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/workloadendpoint)
+`NetworkPolicy` is a namespaced resource. `NetworkPolicy` in a specific namespace
+only applies to [`WorkloadEndpoints`]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/workloadendpoint)
 in that namespace. Two resources are in the same namespace if the namespace
 value is set the same on both.
-See [GlobalNetworkPolicy]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/globalnetworkpolicy) for non-namespaced Network Policy.
+See [`GlobalNetworkPolicy`]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/globalnetworkpolicy) for non-namespaced network policy.
```
Comment (gunjan5) [inline v3.0/reference/calicoctl/resources/networkpolicy.md]: does this render correctly on the site when you use `` for a link?

Comment (emanic) [reply v3.0/reference/calicoctl/resources/networkpolicy.md]: It's a good point....the rendering is not good in this case. It's not obvious that it's a link. I've adjusted this.

File: master/reference/calicoctl/resources/globalnetworkpolicy.md [hunk 4]
```diff
@@ -6,13 +6,13 @@ no_canonical: true
...
-Select a namespace in a GlobalNetworkPolicy in the standard selector by using
-`projectcalico.org/namespace` as the label name and a namespace name as the
+`GlobalNetworkPolicy` is not a namespaced resource. `GlobalNetworkPolicy` applies to [workload endpoints]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/workloadendpoint) in all namespaces, and to [host endpoints]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/hostendpoint).
+Select a namespace in a `GlobalNetworkPolicy` in the standard selector by using
+`projectcalico.org/namespace` as the label name and a `namespace` name as the
 value to compare against, e.g., `projectcalico.org/namespace == "default"`.
-See [NetworkPolicy]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/networkpolicy) for namespaced Network Policy.
+See [network policy]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/networkpolicy) for namespaced network policy.
```
Comment (gunjan5) [inline master/reference/calicoctl/resources/globalnetworkpolicy.md]: Hmm we're talking about `NetworkPolicy` the Calico resource here, not a generic `network policy` (word - which could mean any network policy k8s, calico or the generic word), so I prefer the original syntax. Thanks how other resource docs are also written. Maybe we should create a separate  PR to clean this up for all the resources (including backticks)?

Comment (emanic) [reply master/reference/calicoctl/resources/globalnetworkpolicy.md]: Actually, Kubernetes has a `NetworkPolicy`.

Comment (emanic) [reply master/reference/calicoctl/resources/globalnetworkpolicy.md]: I can do in a different PR I guess or else check now and see.

**Comment by emanic** [comment]
:cyclone:

**Pull request merged**